### DISCUSSION TITLE
Show or move validation message on product page to custom container

### DISCUSSION THREAD

**Discussion by gfxpixeldesigns** [breezefront/community#236] [1. Help] [answered]
I'm trying to move the quantity error validation message which shows when you enter the wrong qty amount on the product page. For example if you enter 1 when minimum quantity increments are enabled you will get the error "You can buy this product only in quantities of 6 at a time."

On Luma i would usually do this by setting the data validation attribute

`data-errors-message-box=`

```
                      <input data-bind="value: qty()" type="number"
                       name="qty"
                       id="qty"
                        maxlength="10"	
                       min="0"
                       value="<?= $block->getProductDefaultQty() * 1 ?>"
                       title="<?= $block->escapeHtmlAttr(__('Qty')) ?>"
                       class="input-text qty"
                       data-validate="<?= $block->escapeHtml(json_encode($block->getQuantityValidators())) ?>"
                       data-errors-message-box="#validation-message-box-simple-product"
                       />
```

And then place an empty div with the same ID as above

`            <div id="validation-message-box-simple-product"></div>`

What would be the correct way of doing this with breeze please? I checked the layout files but couldnt see anything being overriden.

**Comment by vovayatsyuk** [comment]
Oh, didn't know about that feature. Thanks. 

I think I can integrate it in the next release. For now, you can listen to `validateAfter` event:

```js
$(form).on('validateAfter, function () {
    // move error to proper place
});
```

**Comment by vovayatsyuk** [comment, marked as answer]
Done in development branch: https://github.com/breezefront/module-breeze/commit/6a1d451254f7c1db10fa7fec69ab57ec0fc52853

Will be shipped in the next release!
